sentimental/src/main/webapp/sentiment.js

```diff
@@ -0,0 +1,13 @@
+async function sentimentText() {
```

Comment:
Can't this function go in the scripts.js file? Unless there's a specific reason why it can't be in there, I think it would be better to have them all in one file, no?

Reply:
Actually, I just realized that you DO have to put it in scripts.js or we'll have to specify that our javascript is in sentiment.js

Reply:
I do not agree with this and also I do not see a problem to have to reference this file, actually I prefer to have functions separate, it would be easier in the future to correct or change them if it is necessary.

Reply:
Hmm if you think it would be better then okay! I'm just trying to test your code and I can't get it to work so I thought maybe that was the problem. But I'm looking at your code and it seems like it should work, so I think it's an issue on my part :(